apps/drec-ui/src/organization/utils/businessType.ts

```diff
@@ -0,0 +1,7 @@
+import { IRECBusinessLegalStatusLabelsMap } from '@energyweb/utils-general';
+
+export const formatOrganizationBusinessType = (businessType: string): string => {
```

Comment:
Here should be some comment left, since I am not sure that we will user I-REC business statuses later on